src/main/java/controller/LottoController.java

```diff
@@ -1,31 +1,40 @@
 package controller;
 
 import model.*;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class LottoController {
-    private LottoMachine lottoMachine;
+    private final LottoMachine lottoMachine = new LottoMachineImpl();
     private List<UserLotto> userLottos;
 
-    public LottoController(int purchaseAmount, LottoMachineImpl lottoMachine) {
-        this.lottoMachine = lottoMachine;
-        userLottos = generateLotto(purchaseAmount);
+    public LottoController(int purchaseAmount, List<String> inputManualLottos) {
+        userLottos = generateManualUserLottos(inputManualLottos);
+        userLottos.addAll(generateAutoUserLotto(purchaseAmount));
     }
 
-    public LottoController(List<UserLotto> userLottoList, LottoMachineImpl lottoMachine) {
-        userLottos = userLottoList;
-        this.lottoMachine = lottoMachine;
+    private List<UserLotto> generateManualUserLottos(List<String> inputManualLottos) {
+        return inputManualLottos.stream()
+                .map(UserLotto::create)
+                .collect(Collectors.toList());
     }
 
-    private List<UserLotto> generateLotto(int purchaseAmount) {
+    private List<UserLotto> generateAutoUserLotto(int purchaseAmount) {
         return Stream.generate(lottoMachine::extractLottoNumbers)
                 .limit(purchaseAmount)
                 .collect(Collectors.toList());
     }
 
+    public List<List<Integer>> getUserLottosForOutput() {
+        return userLottos.stream()
+                .map(o -> new ArrayList<>(o.getLottoBallNumbers()))
+                .collect(Collectors.toList());
+    }
+
     public List<UserLotto> getUserLottos() {
         return userLottos;
```

Comment:
```suggestion
        return new ArrayList<>(userLottos);
```

이렇게 표현해 볼 수 있을 것 같네요